app/Http/Controllers/KYCController.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\KYC;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+use Auth;
+
+class KYCController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $kycs = KYC::paginate(25);
+        return view('kyc.admin-dashboard', compact('kycs'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $kyc = Auth::user()->kyc;
+        if (Auth::user()->has('kyc') || $kyc->status == 'pending') {
+            return view('kyc.kyc-verify');
+        }
+        return view('kyc.kyc-verifcation');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $request->validate([
+            'fullname' => ['required', 'string', 'max:255'],
+            'photo' => ['max:2048|mimes:jpeg,png,'],
```

Comment:
Change the rules for `photo` and `identification` to this `['required', 'image', 'mimes:png,jpg', 'max:2048']`